src/coreComponents/mesh/EmbeddedSurfaceSubRegion.hpp

```diff
@@ -259,8 +274,40 @@ class EmbeddedSurfaceSubRegion : public SurfaceElementSubRegion
    */
   array1d< real64 > const & getConnectivityIndex() const { return m_connectivityIndex;}
 
-  ///@}
 
+  /**
+   * @brief Get a mutable total displacement array.
+   * @return the total displacement array if it exists, or an error is thrown if it does not exist
+   * @note An error is thrown if the displacement jump does not exist
+   */
+  array2d< real64 > & displacementJump()
+  { return getReference< array2d< real64 > >( viewKeys.dispJump ); }
+
+  /**
+   * @brief Provide an immutable arrayView to the total displacement array.
+   * @return immutable arrayView of the total displacement array if it exists, or an error is thrown if it does not exist
+   * @note An error is thrown if the displacement jump does not exist
+   */
+  arrayView2d< real64 const > displacementJump() const
+  {return getReference< array2d< real64 > >( viewKeys.dispJump ); }
+
+  /**
+   * @brief Get a mutable incremental displacement array.
+   * @return the incremental displacement array if it exists, or an error is thrown if it does not exist
+   * @note An error is thrown if the incremental displacement jump does not exist
+   */
+  array2d< real64 > & incrementalDisplacementJump()
+  { return getReference< array2d< real64 > >( viewKeys.deltaDispJump ); }
+
+  /**
+   * @brief Provide an immutable arrayView to the incremental displacement jump array.
+   * @return immutable arrayView of the incremental displacement array if it exists, or an error is thrown if it does not exist
+   * @note An error is thrown if the incremental displacement jump does not exist
+   */
+  arrayView2d< real64 const > incrementalDisplacementJump() const
+  { return getReference< array2d< real64 > >( viewKeys.deltaDispJump ); }
```

Comment:
If these always exist in the subregion, then can just be members.

Reply:
Well they only exist for mechanics simulations. In case of flow sim they won't be registered.

Reply:
Isn't the displacement jump akin to the aperture?

Reply:
it is but they are two different things. In a way the normal component of it is equal to the aperture but they are two different fields. Basically the flow solver will ask for the aperture which, in the poroelastic case is updated based on the displacement jump. It's a formality but distinguishing between them allows for a bit more flexibility, 
e.g., the enrichment can be done in a basis which does not use the normal to the fracture.